<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormVistaEmotiBit</class>
 <widget class="QWidget" name="FormVistaEmotiBit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1236</width>
    <height>845</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButtonNota">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>580</y>
     <width>93</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Enviar Nota 
a EmotiBit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonConectar">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Conectar Pulsera</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowserPaquetes">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>670</y>
     <width>531</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>580</y>
     <width>311</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>Estado del Dispositivo EmotiBit</string>
   </property>
   <widget class="QLabel" name="label_Grabar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>No Grabando en tarjeta SD</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_PowerStatus">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Power Status:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelBateria">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>129</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Bateria:</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBarBatery">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="value">
     <number>10</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_nombreArchivo">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>--</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Archivo:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Modo">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>--</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; vertical-align:super;&quot;&gt;Los datos que se mestran son enviados desde el dispositivo EmotiBit y&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; vertical-align:super;&quot;&gt;recogidos por la aplicación.   Representan el estado de la pulsera.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="checkBoxPaquetesRecibidos">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>640</y>
     <width>531</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Ver Datos Recibidos desde EmotiBit :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditNota">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>600</y>
     <width>421</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxDispositivos">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>341</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>580</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Nota:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>341</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Messajes de Estado de la Conexión con Dispositivo Emotibit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Dispositivos Detectado</string>
   </property>
  </widget>
  <widget class="FormPlot" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>851</width>
     <height>561</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>241</red>
         <green>244</green>
         <blue>242</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>236</red>
         <green>239</green>
         <blue>237</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Dark">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>85</red>
         <green>255</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Mid">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>127</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Shadow">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="SolidPattern">
        <color alpha="128">
         <red>252</red>
         <green>255</green>
         <blue>252</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>241</red>
         <green>244</green>
         <blue>242</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>252</red>
         <green>255</green>
         <blue>252</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Dark">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>85</red>
         <green>255</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Mid">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>127</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Shadow">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="SolidPattern">
        <color alpha="128">
         <red>252</red>
         <green>255</green>
         <blue>252</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>241</red>
         <green>244</green>
         <blue>242</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Dark">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>85</red>
         <green>255</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Mid">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>127</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Shadow">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="SolidPattern">
        <color alpha="128">
         <red>252</red>
         <green>255</green>
         <blue>252</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>341</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Test de Grabación</string>
   </property>
   <widget class="QPushButton" name="pushButtonStop">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGrabar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Grabar</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxGrabarEnSD">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>211</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Grabar en Tarjeta SD de EmotiBit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; vertical-align:sub;&quot;&gt;Activado grabará  en el el sistema  y en EmotiBit&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QTextBrowser" name="textBrowserMensajes">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>341</width>
     <height>541</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDetectarPulsera">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Detectar Pulsera</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDesconectar">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Desconectar Pulsera</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_kk">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>640</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FormPlot</class>
   <extends>QWidget</extends>
   <header>formplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
